step/sealed-classes.md

```diff
@@ -27,14 +27,14 @@ Additionally a sealed class imposes three constraints on its permitted subclasse
 
 * Every permitted subclass must choose a modifier to describe how it continues the sealing initiated by its superclass:
 
-    * A permitted subclass may be declared final to prevent its part of the class hierarchy from being extended further.
+    * A permitted subclass may be declared final to prevent its part of the class hierarchy from being extended further. It can also be implicitly final (enumerated types and records).
     * A permitted subclass may be declared sealed to allow its part of the hierarchy to be extended further than envisaged by its sealed superclass, but in a restricted fashion.
     * A permitted subclass may be declared non-sealed so that its part of the hierarchy reverts to being open for extension by unknown subclasses. (A sealed class cannot prevent its permitted subclasses from doing this.)
  
 
 ## Your first Sealed Classes
 
-💡 **Sealed Classes** is a preview feature in JDK 16 so make sure that preview features are enabled both at compile time and run-time.
+💡 **Sealed Classes** is a feature in JDK 17.
```

Comment:
we tend to say 'standard and permanent feature'

Reply:
Done